includes/class-wc-google-gtag-js.php

```diff
@@ -476,3 +476,10 @@ public function event_tracking_code( $parameters, $selector ) {
 	}
 
 }
+
+add_filter(
+	'woocommerce_gtag_tracker_variable',
+	function( $tracker_var ) {
+		return 'filteredGtag';
+	}
+);
```

Comment:
I assume this was just for testing and not intended to end up in the final PR.

Reply:
🤦 Thanks for picking up on that @mikkamp that was absolutely a mistake. I've removed the stray filter 👍